### PR TITLE
docs: add v1.24 blog post

### PR DESCRIPTION
**Pull request by iamhopaul123**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/blogs/release-v124.en.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+title: 'AWS Copilot v1.24: ECS Service Connect is coming!'
```
Comment (dannyrandall) [inline site/content/blogs/release-v124.en.md]:
```suggestion
title: 'AWS Copilot v1.24: ECS Service Connect!'
```

Comment (dannyrandall) [reply site/content/blogs/release-v124.en.md]: i feel like when we release it's already _here_

File: site/content/blogs/release-v124.en.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+- **Add `--no-rollback` flag to `env deploy`**: Copilot `env deploy` now has a new flag `--no-rollback`; you can specify the flag to disable automatic env deployment rollback to help with debugging.
+- **Config autoscaling for Request-Driven Web Service**: It is now possible to specify autoscaling configuration for your RDWS. For example, this can be configured in your service manifest:
+```yaml
+ count: high-availability/3
+```
+- **Add log retention to VPC flow logs**: The default value for VPC flow logs is now 14 days (2 weeks).
+ ```yaml
+ network:
+  vpc:
+    flow_logs: on
```
Comment (dannyrandall) [inline site/content/blogs/release-v124.en.md]:
```suggestion
network:
  vpc:
    flow_logs: on
```

File: site/content/blogs/release-v124.en.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+Our public [сommunity сhat](https://gitter.im/aws/copilot-cli) is growing and has over 350 people online and nearly 2.5k stars on [GitHub](http://github.com/aws/copilot-cli/).
+Thanks to every one of you who shows love and support for AWS Copilot.
+
+Copilot v1.24 brings several new features and improvements:
+
+- **ECS Service Connect support**: [See detailed section](#ecs-service-connect-support).
+- **Add `--no-rollback` flag to `env deploy`**: Copilot `env deploy` now has a new flag `--no-rollback`; you can specify the flag to disable automatic env deployment rollback to help with debugging.
+- **Config autoscaling for Request-Driven Web Service**: It is now possible to specify autoscaling configuration for your RDWS. For example, this can be configured in your service manifest:
+```yaml
+ count: high-availability/3
```
Comment (dannyrandall) [inline site/content/blogs/release-v124.en.md]:
```suggestion
count: high-availability/3
```

File: site/content/blogs/release-v124.en.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+```
+- **Add log retention to VPC flow logs**: The default value for VPC flow logs is now 14 days (2 weeks).
```
Comment (dannyrandall) [inline site/content/blogs/release-v124.en.md]:
```suggestion
- **Add log retention to VPC flow logs**: The default value for VPC flow logs is now 14 days.
```

File: site/content/blogs/release-v124.en.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+ network:
+  vpc:
+    flow_logs: on
+ ```
+ Alternatively, you can customize the number of days fore retention:
+ ```yaml
+ network:
+   vpc:
+     flow_logs:
+       retention: 30
```
Comment (dannyrandall) [inline site/content/blogs/release-v124.en.md]:
```suggestion
network:
  vpc:
    flow_logs:
      retention: 30
```

File: site/content/blogs/release-v124.en.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+### Set up Service Connect
+In addition to Service Discovery, you can set up Service Connect with the following configuration in your service manifest.
+
+```yaml
+network:
+  connect: true
+```
+
+Note that in order to use Service Connect, both server and client services need to have Service Connect enabled.
```
Comment (dannyrandall) [inline site/content/blogs/release-v124.en.md]:
```
!!! attention
```
maybe? or info/warn, not sure what the options are

File: site/content/docs/developing/svc-to-svc-communication.en.md [hunk 7]
```diff
@@ -4,13 +4,26 @@
 
 [ECS Service Connect](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-connect.html) enables a client service to connect to its downstream services in a load-balanced and resilient fashion. Furthermore, it simplifies the way of exposing a service to its clients by specifying friendly aliases. With Service Connect in Copilot, each service you create is given the following private alias by default: `http://<your service name>`.
 
+!!! attention
+    Service Connect is not supported yet for our [Request-Driven Web Service](../docs/concepts/services.en.md#request-driven-web-service).
```
Comment (dannyrandall) [inline site/content/docs/developing/svc-to-svc-communication.en.md]:
```suggestion
    Service Connect is not supported yet for [Request-Driven Web Services](../docs/concepts/services.en.md#request-driven-web-service).
```

File: site/content/blogs/release-v124.en.md [hunk 8]
```diff
@@ -21,20 +21,20 @@ Copilot v1.24 brings several new features and improvements:
 - **Add `--no-rollback` flag to `env deploy`**: Copilot `env deploy` now has a new flag `--no-rollback`; you can specify the flag to disable automatic env deployment rollback to help with debugging.
 - **Config autoscaling for Request-Driven Web Service**: It is now possible to specify autoscaling configuration for your RDWS. For example, this can be configured in your service manifest:
 ```yaml
- count: high-availability/3
+count: high-availability/3
 ```
-- **Add log retention to VPC flow logs**: The default value for VPC flow logs is now 14 days (2 weeks).
+- **Add log retention to VPC flow logs**: The default value for VPC flow logs is now 14 days.
```
Comment (huanjani) [inline site/content/blogs/release-v124.en.md]: Nit: this sounds like the default existed before but has changed.
```suggestion
- **Add log retention to VPC flow logs**: There is now a default value of 14 days.
```

File: site/content/blogs/release-v124.en.md [hunk 9]
```diff
@@ -62,7 +62,8 @@ network:
   connect: true
 ```
 
-Note that in order to use Service Connect, both server and client services need to have Service Connect enabled.
+!!! attention
+    Note that in order to use Service Connect, both server and client services need to have Service Connect enabled.
```
Comment (huanjani) [inline site/content/blogs/release-v124.en.md]:
```suggestion
    In order to use Service Connect, both server and client services need to have Service Connect enabled.
```

**Review by huanjani — COMMENTED** [review]
couple of tiny things! otherwise looks good!

File: site/content/blogs/release-v124.en.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+```yaml
+count: high-availability/3
+```
+- **Add log retention to VPC flow logs**: There is now a default value of 14 days.
+ ```yaml
+network:
+  vpc:
+    flow_logs: on
+ ```
+ Alternatively, you can customize the number of days fore retention:
```
Comment (huanjani) [inline site/content/blogs/release-v124.en.md]:
```suggestion
 Alternatively, you can customize the number of days for retention:
```

File: site/content/blogs/release-v124.en.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+    The AWS Copilot CLI is a tool for developers to build, release, and operate production ready containerized applications on AWS.
+    From getting started, pushing to staging, and releasing to production, Copilot can help manage the entire lifecycle of your application development.
+    At the foundation of Copilot is AWS CloudFormation, which enables you to provision infrastructure as code.
+    Copilot provides pre-defined CloudFormation templates and user-friendly workflows for different types of micro service architectures,
+    enabling you to focus on developing your application, instead of writing deployment scripts.
+
+    See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
+
+## ECS Service Connect Support
+ECS just launched [ECS Service Connect](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-connect.html). With [Copilot support](../docs/developing/svc-to-svc-communication.en.md#service-connect), your private service-to-service communication will be more resilient and load balanced compared with using Service Discovery. I will walk you through how Copilot supports ECS Service Connect.
```
Comment (huanjani) [inline site/content/blogs/release-v124.en.md]:
```suggestion
ECS just launched [ECS Service Connect](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-connect.html). With [Copilot support](../docs/developing/svc-to-svc-communication.en.md#service-connect)! Your private service-to-service communication will be more resilient and load-balanced, compared with that when using Service Discovery. Let's walk through how Copilot supports ECS Service Connect.
```

File: site/content/blogs/release-v124.en.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+
+```yaml
+network:
+  connect: true
+```
+
+!!! attention
+    In order to use Service Connect, both server and client services need to have Service Connect enabled.
+
+### Check out available endpoint
```
Comment (huanjani) [inline site/content/blogs/release-v124.en.md]:
```suggestion
### Check out the generated endpoint
```

File: site/content/blogs/release-v124.en.md [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+...
+Internal Service Endpoint
+
+  Endpoint                      Environment  Type
+  --------                      -----------  ----
+  front-end:80                  test         Service Connect
+  front-end.test.demo.local:80  test         Service Discovery
+...
+```
+As shown above, `front-end:80` is your Service Connect endpoint that the other client services can call (must have Service Connect enabled as well).
```
Comment (huanjani) [inline site/content/blogs/release-v124.en.md]:
```suggestion
As shown above, `front-end:80` is your Service Connect endpoint that your other client services can call. (They must have Service Connect enabled as well.)
```

File: site/content/blogs/release-v124.en.md [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+
+### (Optional) Verify it works
```
Comment (huanjani) [inline site/content/blogs/release-v124.en.md]:
```suggestion
### (Optional) Verify that it works
```

File: site/content/blogs/release-v124.en.md [hunk 15]
```diff
@@ -0,0 +1,110 @@
...
+To verify the IP address for your Service Connect endpoint URL is indeed added to your service network, you can simply use `copilot svc exec` to execute into it and check out the hosts file.
```
Comment (huanjani) [inline site/content/blogs/release-v124.en.md]:
```suggestion
To verify that the IP address for your Service Connect endpoint URL has indeed been added to your service network, you can simply use `copilot svc exec` to execute into your container and check out the hosts file.
```

File: site/content/blogs/release-v124.en.md [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+Exiting session with sessionId: ecs-execute-command-088d464a5721fuej3f.
+```
+
+## What’s next?
+
+Download the new Copilot CLI version by following the link below and leave your feedback on [GitHub](https://github.com/aws/copilot-cli/) or our [Community Chat](https://gitter.im/aws/copilot-cli):
+
+- Download [the latest CLI version](../docs/getting-started/install.en.md)
+- Try our [Getting Started Guide](../docs/getting-started/first-app-tutorial.en.md)
+- Read full release notes on [GitHub](https://github.com/aws/copilot-cli/releases/tag/v1.23.0)
```
Comment (huanjani) [inline site/content/blogs/release-v124.en.md]:
```suggestion
- Read full release notes on [GitHub](https://github.com/aws/copilot-cli/releases/tag/v1.24.0)
```

File: site/content/docs/developing/svc-to-svc-communication.en.md [hunk 17]
```diff
@@ -4,13 +4,26 @@
 
 [ECS Service Connect](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-connect.html) enables a client service to connect to its downstream services in a load-balanced and resilient fashion. Furthermore, it simplifies the way of exposing a service to its clients by specifying friendly aliases. With Service Connect in Copilot, each service you create is given the following private alias by default: `http://<your service name>`.
 
+!!! attention
+    Service Connect is not supported yet for [Request-Driven Web Services](../docs/concepts/services.en.md#request-driven-web-service).
```
Comment (huanjani) [inline site/content/docs/developing/svc-to-svc-communication.en.md]:
```suggestion
    Service Connect is not yet supported for [Request-Driven Web Services](../docs/concepts/services.en.md#request-driven-web-service).
```

File: site/content/blogs/release-v124.en.md [hunk 18]
```diff
@@ -0,0 +1,110 @@
...
+image_width: '1051'
+image_height: '747'
+---
+
+# AWS Copilot v1.24: ECS Service Connect!
+
+Posted On: Nov 28, 2022
+
+The AWS Copilot core team is announcing the Copilot v1.24 release.   
+Our public [сommunity сhat](https://gitter.im/aws/copilot-cli) is growing and has over 350 people online and nearly 2.5k stars on [GitHub](http://github.com/aws/copilot-cli/).
```
Comment (huanjani) [inline site/content/blogs/release-v124.en.md]:
```suggestion
Our public [сommunity сhat](https://gitter.im/aws/copilot-cli) is growing and has over 350 people online and over 2.5k stars on [GitHub](http://github.com/aws/copilot-cli/).
```

File: site/content/blogs/release-v124.en.md [hunk 19]
```diff
@@ -0,0 +1,110 @@
...
+    The AWS Copilot CLI is a tool for developers to build, release, and operate production ready containerized applications on AWS.
+    From getting started, pushing to staging, and releasing to production, Copilot can help manage the entire lifecycle of your application development.
+    At the foundation of Copilot is AWS CloudFormation, which enables you to provision infrastructure as code.
+    Copilot provides pre-defined CloudFormation templates and user-friendly workflows for different types of micro service architectures,
+    enabling you to focus on developing your application, instead of writing deployment scripts.
+
+    See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
+
+## ECS Service Connect Support
+ECS just launched [ECS Service Connect](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-connect.html). With [Copilot support](../docs/developing/svc-to-svc-communication.en.md#service-connect)! Your private service-to-service communication will be more resilient and load-balanced, compared with that when using Service Discovery. Let's walk through how Copilot supports ECS Service Connect.
```
Comment (huanjani) [inline site/content/blogs/release-v124.en.md]:
```suggestion
[Copilot supports](../docs/developing/svc-to-svc-communication.en.md#service-connect) the newly launched [ECS Service Connect](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-connect.html)! Your private service-to-service communication will be more resilient and load-balanced with Service Connect than with Service Discovery. Let's walk through how Copilot supports ECS Service Connect.
```

**Review by huanjani — APPROVED** [review]
Sorry I keep suggesting changes! Looks good!

**Review by dannyrandall — APPROVED** [review]

**Pull request merged**